certify-core/src/main/java/io/mosip/certify/core/entity/TemplateData.java

```diff
@@ -0,0 +1,29 @@
+package io.mosip.certify.core.entity;
+
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.*;
+import lombok.*;
+import jakarta.validation.constraints.NotBlank;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+@Entity
+@NoArgsConstructor
+@IdClass(TemplateId.class)
+public class TemplateData {
+    @NotBlank(message = "Template is mandatory")
```

Comment:
let's add created date and updated datetimes

format

@Column(name = "cr_dtimes")
    private LocalDateTime createdtimes;

    @Column(name = "upd_dtimes")
    private LocalDateTime updatedtimes;